floater/generators.py

```diff
@@ -24,9 +24,9 @@ def __init__(self, xlim, ylim, dx=1., dy=1.):
         self.ylim = ylim
         self.Lx = xlim[1] - xlim[0]
         self.Ly = ylim[1] - ylim[0]
-        if (self.Lx % dx) != 0.0:
+        if ((self.Lx*10.0**4.0) % (dx*10.0**4.0))/10.0**4.0 != 0.0:
             raise ValueError("Lx is not divisible evenly by dx")
-        if (self.Ly % dy) != 0.0:
+        if ((self.Ly*10.0**4.0) % (dy*10.0**4.0))/10.0**4.0 != 0.0:
```

Comment:
I guess you are just multiplying everything by a big number...is this a robust solution?